modules/oe_whitelabel_search/src/Plugin/Block/SearchBlock.php

```diff
@@ -249,17 +241,8 @@ public function blockSubmit($form, FormStateInterface $form_state): void {
   public function blockValidate($form, FormStateInterface $form_state): void {
     $values = $form_state->getValues();
 
-    if ($values['input']['input_classes'] !== Html::cleanCssIdentifier($values['input']['input_classes'])) {
-      $form_state->setErrorByName('input][input_classes', $this->t('Field "@field_name" does not contain a valid css class.', [
-        '@field_name' => $form['input']['input_classes']['#title'],
-      ]));
-    }
-
-    if ($values['button']['button_classes'] !== Html::cleanCssIdentifier($values['button']['button_classes'])) {
-      $form_state->setErrorByName('button][button_classes', $this->t('Field "@field_name" does not contain a valid css class.', [
-        '@field_name' => $form['button']['button_classes']['#title'],
-      ]));
-    }
+    $completeForm = $form_state->getCompleteForm();
+    $form_state->setValue('region', $completeForm['region']['#value']);
```

Comment:
This can be simplified: https://github.com/openeuropa/oe_whitelabel/commit/c16126c0d12f9b7c3d6d68a9f91d8488b845adbb